examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -98,9 +98,14 @@ def initialize(self, ctx):
                 logger.warning("Missing the index_to_name.json file.")
 
             # ------------------------------- Captum initialization ----------------------------#
-        self.lig = LayerIntegratedGradients(
-            captum_sequence_forward, self.model.bert.embeddings
-        )
+        if self.setup_config["captum_explanation"]:
```

Comment:
Please review how the code can be structured better by moving the captum related items into the [explain_handle](https://github.com/pytorch/serve/blob/master/docs/custom_service.md#captum-explanations-for-custom-handler)

Reply:
Moved the Captum initialization to get_insight function in the handler, this function get called by explain_handler in the base_handler.py.